Algorithm.CSharp/Alphas/MeanReversionLunchBreakAlpha.cs

```diff
@@ -0,0 +1,175 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Algorithm.Framework;
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Data;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Indicators;
+using QuantConnect.Orders.Fees;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuantConnect.Algorithm.CSharp.Alphas
+{
+    /// <summary>
+    /// This alpha aims to capture the mean-reversion effect of ETFs during lunch-break by ranking 20 ETFs
+    /// on their return between the close of the previous day to 12:00 the day after and predicting mean-reversion 
+    /// in price during lunch-break.
+
+    /// Source:  Lunina, V. (June 2011). The Intraday Dynamics of Stock Returns and Trading Activity: Evidence from OMXS 30 (Master's Essay, Lund University). 
+    /// Retrieved from http://lup.lub.lu.se/luur/download?func=downloadFile&recordOId=1973850&fileOId=1973852
+
+    /// <br><br>This alpha is part of the Benchmark Alpha Series created by QuantConnect which are open sourced so the community and client funds can see an example of an alpha. 
+    /// You can read the source code for this alpha on Github in <a href="https://github.com/QuantConnect/Lean/blob/master/Algorithm.CSharp/Alphas/MeanReversionLunchBreakAlpha.cs">C#</a>
+    /// or <a href="https://github.com/QuantConnect/Lean/blob/master/Algorithm.Python/Alphas/MeanReversionLunchBreakAlpha.py">Python</a>.
+    /// </summary>
+    public class MeanReversionLunchBreakAlpha : QCAlgorithmFramework
+    {
+        public override void Initialize()
+        {
+            SetStartDate(2018, 1, 1);
+
+            SetCash(100000);
+
+            // Set zero transaction fees
+            SetSecurityInitializer(security => security.FeeModel = new ConstantFeeModel(0));
+
+            // Use Hourly Data For Simplicity
+            UniverseSettings.Resolution = Resolution.Hour;
+            SetUniverseSelection(new CoarseFundamentalUniverseSelectionModel(CoarseSelectionFunction));
+
+            // Use MeanReversionLunchBreakAlphaModel to establish insights
+            SetAlpha(new MeanReversionLunchBreakAlphaModel());
+
+            // Equally weigh securities in portfolio, based on insights
+            SetPortfolioConstruction(new EqualWeightingPortfolioConstructionModel());
+        }
+
+        /// <summary>
+        /// Sort the data by daily dollar volume and take the top '20' ETFs
+        /// </summary>
+        private IEnumerable<Symbol> CoarseSelectionFunction(IEnumerable<CoarseFundamental> coarse)
+        {
+            return (from cf in coarse
+                    where !cf.HasFundamentalData
```

Comment:
Good question! In the paper that inspires this algorithm, the researcher used index constituents, therefore only stocks. @HalldorAndersen can confirm this.
I used negative `DollarVolume` instead of the `descending` keyword.

Reply:
We are using ETFs in this strategy. We exclude stocks by ignoring `cf `that have fundamental data. Also, we the most traded ETFs, hence sort by `DollarVolume`.